.github/test_complex.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+./input_files/get-input-files.sh
+export PETSC_ARCH=complex-opt-$COMPILERS-$PETSCVERSION
```

Comment:
This line is actually not needed, since ADflow binary is linked to the correct PETSc library at compile time.

Reply:
Maybe we still need this?

Reply:
Yep I guess I was wrong, my bad.

Reply:
Also, @sseraj would you mind adding the same to Travis? The tests on master are failing because of this erroneous commit I made a while ago to the master branch which removed the environment variable.